actors/common/src/main/java/org/sunbird/learner/actors/otp/OTPActor.java

```diff
@@ -105,43 +106,49 @@ private void verifyOTP(Request request) {
     Map<String, Object> otpDetails = otpService.getOTPDetails(type, key);
     if (MapUtils.isEmpty(otpDetails)) {
       ProjectLogger.log(
-          "OTPActor:verifyOTP: Details not found for type = " + type + " key = " + key,
-          LoggerEnum.DEBUG);
+              "OTPActor:verifyOTP: Details not found for type = " + type + " key = " + key,
+              LoggerEnum.DEBUG);

```

Comment:
@amit-tarento can we change LoggerEnum.DEBUG TO info.

Reply:
Why error , its just a message , not error message